<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportView</class>
 <widget class="QDialog" name="ExportView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Экспорт</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="pathEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="choosePathBtn">
       <property name="text">
        <string>Выбрать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="exportTypeBox">
       <property name="title">
        <string>Тип экспорта</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="pdfBtn">
          <property name="text">
           <string>На печать</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="browserBtn">
          <property name="text">
           <string>Формат веб-браузера</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="exportTypeBox_2">
       <property name="title">
        <string>Формат экспорта</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="memoryDatesBtn">
          <property name="text">
           <string>Памятные даты</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="usualEventsBtn">
          <property name="text">
           <string>Обычные события</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="submitExportBtn">
     <property name="text">
      <string>Экспорт</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
